bin/check-rds-pending.rb

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env ruby
+#
+# check-rds-events
+#
+#
+# DESCRIPTION:
+#   This plugin checks rds clusters for critical events.
+#   Due to the number of events types on RDS clusters the check searches for
+#   events containing the text string 'has started' or 'is being'.  These events all have
+#   accompanying completiion events and are impacting events
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: aws-sdk-v1
+#   gem: sensu-plugin
+#
+# USAGE:
+#  ./check-rds-events.rb -r ${you_region} -s ${your_aws_secret_access_key} -a ${your_aws_access_key}
+#
+# NOTES:
+#
+# LICENSE:
+#   Tim Smith <tim@cozy.co>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+class CheckRDSEvents < Sensu::Plugin::Check::CLI
+  option :aws_access_key,
+         short: '-a AWS_ACCESS_KEY',
+         long: '--aws-access-key AWS_ACCESS_KEY',
+         description: "AWS Access Key. Either set ENV['AWS_ACCESS_KEY_ID'] or provide it as an option",
+         default: ENV['AWS_ACCESS_KEY_ID']
+
+  option :aws_secret_access_key,
+         short: '-s AWS_SECRET_ACCESS_KEY',
+         long: '--aws-secret-access-key AWS_SECRET_ACCESS_KEY',
+         description: "AWS Secret Access Key. Either set ENV['AWS_SECRET_ACCESS_KEY'] or provide it as an option",
+         default: ENV['AWS_SECRET_ACCESS_KEY']
+
+  option :aws_region,
+         short: '-r AWS_REGION',
+         long: '--aws-region REGION',
+         description: 'AWS Region (such as eu-west-1).',
+         default: 'us-east-1'
+
+  def run # rubocop:disable AbcSize
+
+    rds = Aws::RDS::Client.new(
+      access_key_id: config[:aws_access_key],
```

Comment:
Instead of passing in the configs here please use Common.  An example is here
https://github.com/sensu-plugins/sensu-plugins-aws/blob/master/bin/check-ebs-snapshots.rb

Common is here
https://github.com/sensu-plugins/sensu-plugins-aws/blob/master/lib/sensu-plugins-aws/common.rb